src/postamble.js

```diff
@@ -43,6 +43,9 @@ if (memoryInitializer) {
     memoryInitializer = Module['locateFile'](memoryInitializer);
   } else if (Module['memoryInitializerPrefixURL']) {
     memoryInitializer = Module['memoryInitializerPrefixURL'] + memoryInitializer;
+  } else if (ENVIRONMENT_IS_NODE) {
+    // Look for .mem file in the same folder as script by default
+    memoryInitializer = __dirname + '/' + memoryInitializer;
```

Comment:
> Maybe I don't understand the idea for Module['documentDirectory'] - what does "document" mean here?
> 
> Perhaps we need something like Module['pwd']?

There `document` refers on the current "page" or "file" being browsed. I.e. in browser [document.URL](https://developer.mozilla.org/en-US/docs/Web/API/Document/URL) gives the location of the current HTML file that specifies the path that one downloads relative URLs to.

Something like `Module['pwd']` sounds distracting, since this is not about current working directory, and this does not relate to C `chdir()` & `getcwd()` functions, but `'pwd'` might make one think it would.